Swashbuckle.AWSApiGateway.Annotations/Swashbuckle.AWSApiGateway.Annotations.csproj

```diff
@@ -26,13 +27,13 @@
       1.0.2- only include document level annotations that are explicitly initialized
       1.0.1- now supports key source, cors, integration and auth annotations
     </PackageReleaseNotes>
-    <Version>2.4.1</Version>
+    <Version>2.5.0</Version>
     <PackageProjectUrl>https://github.com/waxtell/Swashbuckle.AWSApiGateway.Annotations</PackageProjectUrl>
     <RepositoryUrl>https://github.com/waxtell/Swashbuckle.AWSApiGateway.Annotations.git</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
     <OpenApiGenerateDocuments>false</OpenApiGenerateDocuments>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Swashbuckle.AspNetCore" Version="5.0.0" />
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="6.4.0" />

```

Comment:
Interesting...  I could not build the solution until I added
```csharp
<PackageReference Include="Microsoft.AspNetCore.Cors" Version="2.2.0" />
```

Reply:
Ah, strange. I think something to do with the latest Swashbuckle.AspNetCore?

Reply:
I suspect so...  does it build for you without it?

Reply:
No, it doesn't, just confirmed. I've pushed in a change with the dependency.

> Something out of context though, I was looking to add the security block as below for a controller. Can you guide me on how I can do that?
```json
"/api/Countries" : {
      "get" : {
        "responses" : {
          "200" : {
            "description" : "200 response",
            "content" : {
              "application/json" : {
                "schema" : {
                  "$ref" : "#/components/schemas/ArrayOfCountry"
                }
              }
            }
          }
        },
        "security" : [ {
          "testing-authenticator" : [ ]
        } ]
      }
    }
```